operator/pkg/controllers/etcd/etcd.go

```diff
@@ -208,18 +104,17 @@ The last two entries are added for every pod in the cluster
 */
 func etcdServerCertConfig(controlPlane *v1alpha1.ControlPlane) *secrets.Request {
 	return &secrets.Request{
-		Name: etcdServerSecretNameFor(controlPlane.ClusterName()),
-		CertConfig: &pkiutil.CertConfig{
-			Config: &certutil.Config{
-				Usages:       []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth, x509.ExtKeyUsageClientAuth},
-				CommonName:   "etcd",
-				Organization: []string{"kubernetes"},
-				AltNames: certutil.AltNames{
-					DNSNames: append(etcdPodAndHostnames(controlPlane),
-						etcdSvcFQDN(controlPlane.ClusterName(), controlPlane.Namespace),
-						"localhost"),
-					IPs: []net.IP{net.IPv4(127, 0, 0, 1)},
-				},
+		Name:      serverSecretNameFor(controlPlane.ClusterName()),
+		Namespace: controlPlane.NamespaceName(),
```

Comment:
I did a double take when reading this, since I read NamespacedName instead of NamespaceName. I might just use Namespace directly.